extensions/rich_text_components/Tabs/protractor.js

```diff
@@ -37,18 +39,29 @@ var customizeComponent = async function(modal, tabArray) {
 };
 
 var expectComponentDetailsToMatch = async function(elem, tabArray) {
-  var titleElems = elem.element(by.tagName('ul')).all(by.tagName('li'));
+  var titleElems = elem.all(
+    by.css('.protractor-non-interactive-tabs-headers'));
```

Comment:
Can this be changed to use a class of the form `protractor-test-*`?

Reply:
Done!